nautobot/core/views/generic.py

```diff
@@ -966,7 +969,9 @@ def post(self, request, **kwargs):
                 form_custom_fields = getattr(form, "custom_fields", [])
                 form_relationships = getattr(form, "relationships", [])
                 standard_fields = [
-                    field for field in form.fields if field not in form_custom_fields + form_relationships + ["pk"]
+                    field
+                    for field in form.fields
+                    if field not in form_custom_fields + form_relationships + ["pk"] + ["object_note"]
```

Comment:
tagging @HanlinMiao - we'll want to make sure we get equivalent updates into the NautobotViewSet work.